### PR TITLE
Minor tweaks to breakpoint handling

### PR DESCRIPTION
**Pull request by jduck**
* Fix "dbm" help string
* Save <module>+<offset> as the name when using dbm
* Fix allocation patterns for "module"
* Remove commented out code
* White space fixes

### CODE REVIEW

**Comment by radare** [comment]
Personaly i would prefer the unix philosophy of not saying anything if the command succeds

**Comment by jduck** [comment]
Okay, printing removed.

File: libr/core/cmd_debug.c [hunk 1]
```diff
@@ -1973,17 +1972,16 @@ static void r_core_cmd_bp(RCore *core, const char *input) {
...
 					free (bpi->name);
 					if (!strcmp (input + 2, "$$")) {
 						char *newname = NULL;
-						//RFlagItem *f = r_flag_get_at (core->flags, addr);
 						RFlagItem *f = r_flag_get_i2 (core->flags, addr);
+
 						if (f) {
 							if (f->offset != addr) {
-								newname = r_str_newf ("%s+%d\n", f->name, (int)(addr - f->offset));
+								newname = r_str_newf ("%s+0x%" PFMT64x "\n", f->name, (int)(addr - f->offset));
```
Comment (radare) [inline libr/core/cmd_debug.c]: ut64 != int

File: libr/core/cmd_debug.c [hunk 2]
```diff
@@ -1993,8 +1991,7 @@ static void r_core_cmd_bp(RCore *core, const char *input) {
 						bpi->name = strdup (input + 2);
 					}
 				} else {
-					eprintf ("Cannot set breakpoint at "
-						"'%s'\n", input+2);
+					eprintf ("Cannot set breakpoint at '%s'\n", input+2);
```
Comment (radare) [inline libr/core/cmd_debug.c]: add spaces around +

File: libr/debug/debug.c [hunk 3]
```diff
@@ -66,55 +66,59 @@ static int r_debug_recoil(RDebug *dbg) {
 R_API RBreakpointItem *r_debug_bp_add(RDebug *dbg, ut64 addr, int hw, char *module, st64 m_delta) {
 	int bpsz = strcmp (dbg->arch, "arm") ? 1 : 4;
 	RBreakpointItem *bpi;
+	char *module_name = module;
```
Comment (radare) [inline libr/debug/debug.c]: maybe const char *?

**Comment by jduck** [comment]
I've updated the PR addressing your comments.

File: libr/core/cmd_debug.c [hunk 4]
```diff
@@ -1973,17 +1972,16 @@ static void r_core_cmd_bp(RCore *core, const char *input) {
...
 					free (bpi->name);
 					if (!strcmp (input + 2, "$$")) {
 						char *newname = NULL;
-						//RFlagItem *f = r_flag_get_at (core->flags, addr);
 						RFlagItem *f = r_flag_get_i2 (core->flags, addr);
+
 						if (f) {
 							if (f->offset != addr) {
-								newname = r_str_newf ("%s+%d\n", f->name, (int)(addr - f->offset));
+								newname = r_str_newf ("%s+0x%" PFMT64x "\n", f->name, addr - f->offset);
```
Comment (radare) [inline libr/core/cmd_debug.c]: addr should be always bigger than f->offset.maybe this if should be if (f->offset < addr) to ensure this condition at runtime

**Comment by radare** [comment]
i have fixed this, squashed and pushed.

thanks

**Comment by jduck** [comment]
OH ok. Thanks!

**Pull request closed without merging**